bin/aws/ssm_consul_server.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SERVERS=$(aws ec2 describe-instances --filter Name=tag:Name,Values="Consul Server")
+CONSUL_SERVER_INSTANCE_ID=$(echo "${SERVERS}" | jq -r .Reservations[0].Instances[0].InstanceId)
```

Comment:
You could shorten this to a single command if you like:

```sh
aws ec2 describe-instances \
    --filter Name=tag:Name,Values="Consul Server" \
    --query="Reservations[0].Instances[0].InstanceId" \
    --output=text
```
The AWS CLI uses [JMESPath](https://jmespath.org/) for the query language, which is similar to (but not the same as!) `jq`, and can be useful. Here, the main value would be simplifying the Bash (not that it's particularly complicated Bash, of course, but less Bash is often better 😅).